pkg/argocd/update.go

```diff
@@ -449,6 +450,10 @@ func parseGitConfig(app *v1alpha1.Application, kubeClient *kube.KubernetesClient
 	if ok {
 		wbc.GitBranch = strings.TrimSpace(branch)
 	}
+	writebranch, ok := app.Annotations[common.GitWriteBranchAnnotation]
+	if ok {
+		wbc.GitWriteBranch = strings.TrimSpace(writebranch)
+	}
```

Comment:
Also, instead of accessing annotations directly, can you please add an appropriate method to get the value from the app to [here](https://github.com/argoproj-labs/argocd-image-updater/blob/master/pkg/image/options.go)

Reply:
The file you linked is for per-image configuration? It looks like all the app-level config (like all the existing code for `WriteBackConfig` is already accessed in this way.

Reply:
Yeah, you're right - my fault, sorry. 

I should refactor the code to encapsulate the global options into getter methods as well some day :) Never mind then - and with encoding the write back branch in the same annotation as the checkout branch, this change is also not necessary anymore.